From: Mauro Carvalho Chehab <mchehab@redhat.com>
Date: Tue, 23 Nov 2010 17:33:40 -0500
Subject: [edac] i7core_edac: move static vars to the top of file
Message-id: <20101123153340.66346304@pedra>
Patchwork-id: 29557
O-Subject: [PATCH RHEL5 01/29] BZ#:651869 i7core_edac: move static vars to the
	beginning of the file
Bugzilla: 651869

Changeset: 18c29002f95bc2f67c1c78d7fc7932843aa66657

While here, don't initialize probed with 0.

Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/edac/i7core_edac.c b/drivers/edac/i7core_edac.c
index 5ea848d..5292f0a 100644
--- a/drivers/edac/i7core_edac.c
+++ b/drivers/edac/i7core_edac.c
@@ -39,6 +39,11 @@
 
 #include "edac_mc.h"
 
+/* Static vars */
+static LIST_HEAD(i7core_edac_list);
+static DEFINE_MUTEX(i7core_edac_lock);
+static int probed;
+
 /*
  * This is used for Nehalem-EP and Nehalem-EX devices, where the non-core
  * registers start at bus 255, and are not reported by BIOS.
@@ -263,10 +268,6 @@ struct i7core_pvt {
 	unsigned		mce_overrun;
 };
 
-/* Static vars */
-static LIST_HEAD(i7core_edac_list);
-static DEFINE_MUTEX(i7core_edac_lock);
-
 #define PCI_DESCR(device, function, device_id)	\
 	.dev = (device),			\
 	.func = (function),			\
@@ -1941,8 +1942,6 @@ fail:
  *		< 0 for error code
  */
 
-static int probed = 0;
-
 static int __devinit i7core_probe(struct pci_dev *pdev,
 				  const struct pci_device_id *id)
 {
